Mathlib/CategoryTheory/Monoidal/Cartesian/Grp_.lean

```diff
@@ -47,14 +52,15 @@ def Grp_Class.ofRepresentableBy (F : Cᵒᵖ ⥤ Grp.{w}) (α : (F ⋙ forget _)
 
 /-- If `G` is a group object, then `Hom(X, G)` has a group structure. -/
 abbrev Hom.group : Group (X ⟶ G) where
-  __ := monoid
   inv f := f ≫ ι
   inv_mul_cancel f := calc
     lift (f ≫ ι) f ≫ μ
     _ = (f ≫ lift ι (𝟙 G)) ≫ μ := by simp
     _ = toUnit X ≫ η := by rw [Category.assoc]; simp
 
-attribute [local instance] Hom.group
+scoped[Hom] attribute [instance] Hom.group
```

Comment:
In what namespace does this gets scoped? `open scoped Hom` seems like a very unintuitive thing to open to get that a `Hom`-type is a group. Also here, `Hom.group` is in the root namespace (not even `CategoryTheory`), this feels wrong.

Reply:
IMO this should be scoped in `Mon_Class.Mon`.